source/s3_auto_ranged_put.c

```diff
@@ -510,7 +496,9 @@ static bool s_s3_auto_ranged_put_update(
                 request = aws_s3_request_new(
                     meta_request,
                     AWS_S3_AUTO_RANGED_PUT_REQUEST_TAG_CREATE_MULTIPART_UPLOAD,
-                    0,
+                    AWS_S3_REQUEST_TYPE_CREATE_MULTIPART_UPLOAD,
+                    "CreateMultipartUpload",
```

Comment:
debatable: I would prefer to have a global static string for the operation names we set (I know you are not a fan of those).
- In case the name changed from the model, it's easier to us to update
- multiple places sharing the same string.

Reply:
I've got another idea that will put all the strings in the same place... next push

Reply:
I eliminated pretty much all the repeatedly typed strings
Still not using global static variables 😛